BearlyHumans/squirrelville
# Animation Manager

- Added a data structure in the squirrel controller which uses a list of 'events' to expose animation options in the inspector.
- Added test animation events in the code (moving, idle, jump).

Test:
* Add an event to the Animation Events list in the Squirrel controller.
* Set the trigger to Jump / Moving / Not Moving.
* Set the type and param name to match an animation parameter in the animation controller (e.g. MoveState, int).
* Set the value to the expected value for the event chosen (0 = walk, 1 = move, 2 = jump).
* Play scene, player should be animating.

Bugs/Notes:
> Events are not set up in prefab/scene to avoid conflicts.
> Most events are not triggered in code yet.
